modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -96,7 +163,29 @@ Diagram.defaultProps = {
 			top: '',
 		},
 	},
+	newPinSettings: {
+		colorPicker: {
+			defaultColors: [
+				'AC68D7',
+				'96D470',
+				'F2EE8F',
+				'F4C4A9',
+				'F1A3BB',
+				'67DC19',
+				'959FEF',
+				'A6C198',
+				'FED998',
+				'#38F95',
+				'FD9945',
+				'1A588B',
+			],
+			selectedColor: '0B5FFF',
+			useNative: true,
+		},
+		defaultRadius: 15,
+	},
 	pins: [],
+	spritemap: './assets/clay/icons.svg',
```

Comment:
Will this code run in DXP platform or externally? If you are going to run in DXP, `spritemap` comes automatically from a context called ClayIconSpriteContext wrapped inside `render` from `frontend-js-react-web`

Reply:
in any way we need a spritemap to render icons in ClayIconSpritemapContext or am i wrong? it also tried to remove it but all the icons disappear.
`<ClayIconSpriteContext.Provider value={spritemap}>
      <ClayIcon symbol="add-cell" />
    </ClayIconSpriteContext.Provider>`

Reply:
Did you define `spritemap`, I think you might have to? `const spritemap = './assets/clay/icons.svg'` I think. Never sure myself about the spritemap 🤷